src/ggml-cuda.cu

```diff
@@ -6585,7 +6732,8 @@ static void ggml_cuda_op_mul_mat(
         if (convert_src1_to_q8_1) {
             src1_ddq[id] = (char *) ggml_cuda_pool_malloc(nrows1*src1_padded_col_size*q8_1_ts/q8_1_bs, &src1_asq[id]);
 
-            if (split && src1_on_device && src1_is_contiguous) {
+            // FIXME: why split only? src1 never gets quantized, breaks ggml-backend/GPT-2
+            if (/*split &&*/ src1_on_device && src1_is_contiguous) {
```

Comment:
@JohannesGaessler does this fix look ok? Without it, when src0 is `GGML_BACKEND_GPU` rather than split, src1 is never quantized.

Reply:
It should be fine. This is not a case that I previously considered for the implementation (because for LLaMA the backend for quantized matrices is always `GGML_BACKEND_GPU_SPLIT`) and I don't remember this being a necessity.